app/common/renderer/actions/Inspector.js

```diff
@@ -368,6 +369,23 @@ export function setSessionDetails({driver, sessionDetails, mode, mjpegScreenshot
   };
 }
 
+export function storeSessionSettings(updatedSessionSettings = null) {
+  return async (dispatch, getState) => {
+    let sessionSettings;
+    if (updatedSessionSettings === null) {
+      const action = applyClientMethod({
+        methodName: 'getSettings',
+        skipRefresh: true,
+        ignoreResult: true,
+      });
+      sessionSettings = await action(dispatch, getState);
+    } else {
+      sessionSettings = updatedSessionSettings;
```

Comment:
you may assign the default value upon variable declaration. There is no need for this `else` block